tests/app.js

```diff
@@ -0,0 +1,44 @@
+'use strict'
+
+const Fastify = require('fastify')
+const fp = require('fastify-plugin')
+const fastifyMongodb = require('fastify-mongodb')
+const mongodb = require('mongodb')
+
+function app(fastify, config, done) {
+  fastify.register(fastifyMongodb, { client: config.mongodb.client })
+  fastify.get('/', async function (_req, reply) {
+    const db = this.mongo.client.db()
+    const results = await db.collection('examples').find()
+    return reply.send(results)
+  })
+
+  fastify.addHook('onClose', async () => {
+    await config.mongodb.client.close()
+  })
```

Comment:
In this way, we are skipping the `fastify-mongo` utility.
Why prefer to give the client as input instead of providing to the plugin the mongo URL?